src/cocotb/decorators.py

```diff
@@ -146,39 +190,58 @@ async def my_test(dut):
                 Passing a :class:`bool` value was removed.
                 Pass a specific :class:`Exception` or a tuple of Exceptions instead.
 
-        skip (bool, optional):
+        skip:
             Don't execute this test as part of the regression. Test can still be run
             manually by setting :make:var:`TESTCASE`.
 
-        stage (int)
+        stage:
             Order tests logically into stages, where multiple tests can share a stage.
             Defaults to 0.
+
+        name:
+            Override the default name of the test.
+            The default test name is the ``__qualname__`` of the decorated test function.
```

Comment:
```suggestion
            The default test name is the :any:`__qualname__` of the decorated test function.
```
Will hopefully point to https://docs.python.org/3/library/stdtypes.html?highlight=__qualname__#definition.__qualname__